storage/client.go

```diff
@@ -330,6 +346,47 @@ func NewAccountSASClient(account string, token url.Values, env azure.Environment
 	return c
 }
 
+// NewAccountSASClientFromEndpointToken constructs a client that uses accountSAS authorization
+// for its operations using the specified endpoint and SAS token.
+func NewAccountSASClientFromEndpointToken(endpoint string, sasToken string) (Client, error) {
```

Comment:
>NewAccountSASClientFromEndpointToken [](start = 5, length = 36)

is this doing the same as `NewAccountSASClient`, but from other parameters, right?

Reply:
That's correct.  Take a look [here](https://docs.microsoft.com/en-us/azure/storage/common/storage-dotnet-shared-access-signature-part-1#authenticating-from-a-client-application-with-a-sas) for more details, but the TL;DR version is you can specify and endpoint and SAS token in a connection string; this handles that case when called from ```NewClientFromConnectionString``` and exporting it seemed like a nice thing to do.